source/transfers/contracts/TransferHandlerRegistry.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity 0.5.12;
+
+import "./interfaces/ITransferHandler.sol";
+
+/**
+  * @title TransferHandlerRegistry: holds registry of contract to
+  * facilitate token transfers
+  */
+contract TransferHandlerRegistry {
+
+  event AddTransferHandler(
+    bytes4 kind,
+    address contractAddress
+  );
+
+  event RemoveTransferHandler(
+    bytes4 kind,
+    address contractAddress
+  );
+
+  // Mapping of bytes4 to contract interface type
+  mapping (bytes4 => ITransferHandler) private transferHandlerMapping;
+
+  /**
+  * @notice Adds handler to mapping
+  * @param _kind bytes4
+  * @param _transferHandler ITransferHandler
+  */
+  function addTransferHandler(bytes4 _kind, ITransferHandler _transferHandler)
+    external {
+      transferHandlerMapping[_kind] = _transferHandler;
+      emit AddTransferHandler(_kind, address(_transferHandler));
+    }
+
+  /**
+  * @notice Removes handler from mapping
+  * @param _kind bytes4
+  */
+  function removeTransferHandler(bytes4 _kind)
+    external {
+      delete transferHandlerMapping[_kind];
```

Comment:
Anyone can delete entries? They could stop all trades being possible on our network by removing all handlers and therefore stopping all swaps from being able to occur

Reply:
removed